From c7a936515a2b4232cf3688489ce7c62403f8e15e Mon Sep 17 00:00:00 2001
From: Craig Tatlor <ctatlor97@gmail.com>
Date: Mon, 21 May 2018 17:18:33 +0100
Subject: [PATCH] keymaster: Build with BOARD_VNDK_VERSION

Adds libhardware as a lib and replaces nativebase's UniquePtr as
nativebase isnt a vndk library.

Change-Id: I32691b7543a35ece110d25fe907c181218801755
Signed-off-by: Mohd Faraz <mohd.faraz.abc@gmail.com>
---
 Android.mk | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Android.mk b/Android.mk
index 4d07dd1..9a3b7f2 100644
--- a/Android.mk
+++ b/Android.mk
@@ -37,7 +37,8 @@ LOCAL_SHARED_LIBRARIES := \
         liblog \
         libc \
         libdl \
-        libcutils
+        libcutils \
+        libhardware
 
 LOCAL_HEADER_LIBRARIES := generated_kernel_headers
 
-- 
2.17.1

